benchto-driver/src/test/java/com/teradata/benchto/driver/loader/BenchmarkLoaderTest.java

```diff
@@ -271,12 +271,13 @@ public void getAllBenchmarks_activeVariables_with_regex()
         withActiveVariables("format=(rc)|(tx)");
 
         assertLoadedBenchmarksCount(4).forEach(benchmark ->
-                        assertThat(benchmark.getVariables().get("format")).isIn("orc", "txt")
+                assertThat(benchmark.getVariables().get("format")).isIn("orc", "txt")
         );
     }
 
     @Test
-    public void allBenchmarks_load_only_not_executed_within_two_days() {
+    public void allBenchmarks_load_only_not_executed_within_two_days()
+    {
```

Comment:
why is so? 

I think we need to have a codestyle file added to this project and make sure that codestyle is checked during the build.